AirCasting/Utils/ThresholdSelector.swift

```diff
@@ -0,0 +1,13 @@
+// Created by Lunar on 16/07/2021.
+//
+
+import Foundation
+
+func thresholdFor(thresholds: [SensorThreshold], stream: MeasurementStreamEntity?) -> SensorThreshold? {
+    let match = thresholds.first { threshold in
+        let streamName = stream?.sensorName?
+            .lowercased()
+        return threshold.sensorName?.lowercased() == streamName
+    }
+    return match
```

Comment:
the `match` is not needed at all, you can just leave `thresholds.first { ... }` as the expression type is matching func return type